backend/libs/artemisdb/artemisdb/artemisdb/models.py

```diff
@@ -233,6 +233,13 @@ def in_scope(cls, scopes: list[list[list[str]]]) -> models.QuerySet:
             # No filter was built up (no scopes?) so return nothing
             return cls.objects.none()
 
+    @classmethod
```

Comment:
Should this be a static method since `cls` isn't used/needed?

Reply:
Good catch, but I think it's the other way around. We should use `cls` instead of `Repo` to match the pattern in `in_scope` above